site/includes/nav.pug

```diff
@@ -26,5 +26,5 @@
           span.spectrum-SideNav-link-text Spectrum
       unless process.env.NODE_ENV === 'development'
         li.spectrum-SideNav-item
-          a.spectrum-SideNav-itemLink(href='preview/', rel='noopener')
+          a.spectrum-SideNav-itemLink(href=process.env.CHROMATIC_URL ? process.env.CHROMATIC_URL : 'preview/', rel='noopener')
```

Comment:
So this will update the storybook link from the docs site to point at Chromatic if we have a Chromatic build? Am I understanding that?

Reply:
Exactly yes! Prevent CI from doing a double build of storybook

Reply:
Thanks for this question and also for handling this. This was my big question when I looked at the deploy URL, too.